src/_includes/layouts/demo.njk

```diff
@@ -0,0 +1,23 @@
+{% set demoTitle = title %}
+{% set allDemoContentUpdates = collections.contentUpdates | contentUpdatesOfType('demos') %}
+{% set currentDemoContentUpdates = allDemoContentUpdates | contentUpdatesOfType('demos') %}
+
+<h1>{{ demoTitle }}</h1>
+
+{% if currentDemoContentUpdates.length %}
+  <ul>
+    {% for item in currentDemoContentUpdates | reverse %}
+      <li>
+        {# This line ↓↓ returns the HTML output (what .md files are compiled into) EXACTLY the way we want it!! ↓↓ #}
+        {{ item.templateContent | truncate(100) | safe }}
```

Comment:
This line is key. 🔑 

Figuring this out gave me the most trouble while working on this feature.